crates/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -45,8 +50,10 @@ pub struct PathEnvironment<S> {
     /// The temporary directory that this environment will use during transactions
     pub temp_dir: PathBuf,
 
-    /// The [EnvironmentName] of this environment
-    pub name: EnvironmentName,
+    /// The associated [PathPointer] of this environment.
+    ///
+    /// Used to identify the environment.
+    pub pointer: PathPointer,
```

Comment:
question nonblocking: why store the pointer instead of just the EnvironmentName if all the pointer contains is the name?

Reply:
I could say its to make it easier to extend the file with more metadata, but short term its mainly for consistency with ManagedEnvs.